src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -34,133 +34,96 @@ kubectl config current-context
+New Relic has several charts for the different components which offer different features for the platform:
 
-<CollapserGroup>
-  <Collapser
-      className="freq-link"
-      id="install-helm3"
-      title="Helm 3 using nri-bundle (recommended)"
-  >
-
-    New Relic has several charts for the different components which offer different features for the platform:
-
-    - [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure):
-    Contains the main Kubernetes integration and the infrastructure agent. This is the core component for the New Relic
-    Kubernetes experience.
-    - [`newrelic-logging`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging): Provides a
-    DaemonSet with New Relic's [Fluent Bit](https://fluentbit.io/) output
-    [plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to easily forward your logs to
-    [Logs](https://docs.newrelic.com/docs/logs/new-relic-logs/get-started/introduction-new-relic-logs).
-    - [`nri-kube-events`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-kube-events): Collects and
-    reports cluster events (such as `kubectl get events`) to New Relic.
-    - [`nri-metadata-injection`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-metadata-injection):
-    Automatically injects metadata in applications deployed to the cluster to correlate APM monitoring with
-    Infrastructure monitoring.
-    - [`nri-prometheus`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-prometheus): New Relic's
-    Prometheus OpenMetrics Integration, allows to scrape Prometheus endpoints and [report metrics to New
-    Relic](https://docs.newrelic.com/docs/integrations/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic#OpenMetrics).
-    - [`nri-statsd`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-statsd): New Relic StatsD
-    integration.
-
-    Additionally, New Relic provides
-    [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle), a chart which pulls a
-    selectable set of the charts mentioned above. `nri-bundle` can also install Kube State Metrics for you if needed.
-
-    We recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart
-    for Kubernetes deployments, as it provides the easiest way to select, install, and upgrade individual components at
-    your will.
-
-    ### Installing and configuring `nri-bundle` with Helm
-
-    1. Set the cluster where you want to install the agent:
-
-    ```
-    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
-    ```
+- [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure): Contains the main Kubernetes integration and the infrastructure agent. This is the core component for the New Relic Kubernetes experience.
```

Comment:
should we mention that is responsible for the cluster explorer?

Reply:
Done!